bundle/bundle_test.go

```diff
@@ -613,3 +613,22 @@ func TestBundle_RoundTrip(t *testing.T) {
 		})
 	}
 }
+
+func TestDigestPresent(t *testing.T) {
+	bun, err := ioutil.ReadFile("../testdata/bundles/digest.json")
+	require.NoError(t, err, "couldn't read test bundle")
+
+	bundle, err := Unmarshal(bun)
+	require.NoError(t, err, "the bundle should have been valid")
+
+	//invocationImages[] should have exactly 1 value
+	require.Equal(t, 1, len(bundle.InvocationImages), "there should be one invocation image in the bundle")
+	//the invocation image digest should equal sha256:aaaaaaa...
+	assert.Equal(t, "sha256:aaaaaaa...", bundle.InvocationImages[0].Digest)
+
+	//images[] should have exactly 1 values
+	_, ok := bundle.Images["my-microservice"]
```

Comment:
Why not set the value into a variable here rather than ignoring it. Then the variable can be used in the later check of the digest.

Reply:
Updated.